src/components/pages/Landing/RenderLandingPage.jsx

```diff
@@ -1,11 +1,132 @@
 import React from 'react';
+import WhiteLogo from '../../../styles/Images/WhiteLogo.png';
+import GrantRatesByOfficeImg from '../../../styles/Images/bar-graph-no-text.png';
+import GrantRatesByNationalityImg from '../../../styles/Images/pie-chart-no-text.png';
+import GrantRatesOverTimeImg from '../../../styles/Images/line-graph-no-text.png';
+import HrfPhoto from '../../../styles/Images/HRF2.png';
+import '../../../styles/RenderLandingPage.less';
+import { Button } from 'antd';
 // for the purposes of testing PageNav
 // import PageNav from '../../common/PageNav';
 
 function RenderLandingPage(props) {
+  const scrollToTop = () => {
+    document.body.scrollTop = 0;
+    document.documentElement.scrollTop = 0;
+  };
+
   return (
-    <div>
-      <h1>Place holder Landing page</h1>
+    <div className="main">
+      <div className="header">
+        <div className="hrf-logo-img-container">
```

Comment:
- HRF logo in mobile is too small. it could be reviewed in next iteration to be included in media-queries updates.
-